packages/ra-core/src/controller/edit/useEditController.ts

```diff
@@ -7,7 +7,8 @@ import {
     Record,
     MutationMode,
     OnSuccess,
-    OnFailure,
+    onError,
```

Comment:
A type starting with a lowercase letter? Feels incorrect.

Reply:
And should be removed anyway